### PR TITLE
Sccs for auth 3

### PR DESCRIPTION
**Pull request by maskpa100**
Доработка Log-out , refactor scss

### CODE REVIEW

**Review by Viarcinski — APPROVED** [review]
Looks good to me

File: components/auth/Login/FormFields/EmailFormField.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { Dispatch, SetStateAction } from 'react'
 import style from '../LoginForm.module.scss'
 import { MainInput } from '@/components/common/Inputs/Inputs'
-import { ValidateEmail } from '../validate'
+import { ValidateEmail } from '../../../common/validate/validate'
```
Comment (TURP1) [inline components/auth/Login/FormFields/EmailFormField.tsx:4]: use @/ to imports

File: components/auth/Login/FormFields/PasswordFormField.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import * as Form from '@radix-ui/react-form'
 import style from '../LoginForm.module.scss'
 import { PasswordInput } from '@/components/common/Inputs/Inputs'
-import { ValidatePassword } from '../validate'
+import { ValidatePassword } from '../../../common/validate/validate'
```
Comment (TURP1) [inline components/auth/Login/FormFields/PasswordFormField.tsx:5]: same here

File: styles/_library.scss [hunk 3]
```diff
@@ -1,17 +1,51 @@
...
+$light-500: #EDF3FA;
+$light-700: #D5DAE0;
+$light-900: #8D9094;
+$dark-100: #4C4C4C;
+$dark-300: #333;
+$dark-500: #171717;
+$dark-700: #0D0D0D;
+$dark-900: #000;
 
 @mixin btn {
```
Comment (TURP1) [inline styles/_library.scss:37]: that`s not the place for mixin`s ,thats the place for vars . Create another place where mixin`s for component`s will be

File: components/auth/Login/LoginForm.module.scss [hunk 4]
```diff
@@ -80,12 +80,6 @@
...
       font-size: 13px;
     }
-    .button {
-      border-radius: 2px;
-      background-color: #397df6;
-      color: #fff;
-      margin-top: 14px;
-    }
   }
 }
```
Comment (TURP1) [inline components/auth/Login/LoginForm.module.scss:85]: looks like autoformat working not correctly  , fix it

File: components/auth/Registration/RegistrationForm.module.scss [hunk 5]
```diff
@@ -27,10 +27,8 @@
       }
     }
     h1 {
-      color: #fff;
+      @include s.h1;
```
Comment (TURP1) [inline components/auth/Registration/RegistrationForm.module.scss:30]: your "s" don`t describe itself, use imagination to name it. We are working in team , all of us need to understand what you are including here,  without diving into file

File: components/auth/Registration/RegistrationForm.tsx [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import style from './RegistrationForm.module.scss'
 import { useForm} from 'react-hook-form'
 import { getLayout } from '@/components/Layout/Layout'
-import { ValidateUsername, ValidateEmail, ValidatePassword } from '../Login/validate'
+import { ValidateUsername, ValidateEmail, ValidatePassword } from '../../common/validate/validate'
```
Comment (TURP1) [inline components/auth/Registration/RegistrationForm.tsx:4]: @/

File: package-lock.json
Comment (TURP1) [inline package-lock.json:1]: We agreed not to use npm, delete this file from your`s PR

File: pages/auth/log-out/index.tsx [hunk 7]
```diff
@@ -1,17 +1,35 @@
...
-        localStorage.removeItem('accessToken');
-        router.push('/')
-      }, [])
+      const performLogout = async () => {
+        try {
+          await logout({})
+          .unwrap()
+          .then(() => {
+            router.push('/auth/login')
+            localStorage.removeItem('accessToken');
```
Comment (TURP1) [inline pages/auth/log-out/index.tsx:20]: I`m not sure , that we have to clean accessToken manually, 
I can assume that this is done somehow differently, we need to ask the backend.
It turns out that our request makes no sense if we do it manually

**Review by TURP1 — CHANGES_REQUESTED** [review]

File: pages/auth/log-out/index.tsx [hunk 8]
```diff
@@ -1,17 +1,35 @@
...
+import  { useEffect } from 'react'
 import { useRouter } from 'next/router'
 import { getLayout } from '@/components/Layout/Layout'
 import { useLogoutMutation } from '@/assets/api/auth/authApi'
-import { PageWrapper } from '@/components/PageWrapper/PageWrapper'
-import { Loading } from '@/components/common/loaders/Loading'
+
+
 
 const LogOut = () => {
```
Comment (irinrudko) [inline pages/auth/log-out/index.tsx:8]: rename to **Logout** please

**Review by irinrudko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**